cloudify_rest_client/utils.py

```diff
@@ -28,10 +28,14 @@ def tar_file(file_to_tar, destination_dir, tar_name=''):
     :param tar_name: optional tar name.
     :return:
     """
+    def _reset_tarinfo(tarinfo):
+        tarinfo.mode = 0755
```

Comment:
- What about file ownership? can / should we guarantee `cfyuser:cfyuser`? Maybe `cfyuser:adm`?
- Not sure that `0755` is the right thing to do (depends on the file ownership we choose to go with)

Reply:
it is going to be cfyuser guaranteed, because that's the user that saves the file (restservice)

Reply:
@tehasdf and the group? will be the primary group of `cfyuser`, right?

Reply:
i believe so